fotoapparat/src/main/java/io/fotoapparat/FotoapparatSwitcher.java

```diff
@@ -0,0 +1,77 @@
+package io.fotoapparat;
+
+import android.support.annotation.NonNull;
+
+/**
+ * Switches between different instances of {@link Fotoapparat}. Convenient when you want to allow
+ * user to switch between different cameras or configurations.
+ * <p>
+ * This class is not thread safe. Consider using it from a single thread.
```

Comment:
One should always assume that things are not thread safe. This JavaDoc just confirms it.